source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -0,0 +1,48 @@
+#include "BioXASAppController.h"
+
+BioXASAppController::BioXASAppController(QObject *parent) :
+    AMAppController(parent)
+{
+
+}
+
+BioXASAppController::~BioXASAppController()
+{
+
+}
+
+void BioXASAppController::shutdown()
+{
+	// Make sure we release/clean-up the beamline interface
+	AMBeamline::releaseBl();
+	AMAppController::shutdown();
+}
+
+QGroupBox* BioXASAppController::createSqueezeGroupBoxWithView(QWidget *view, const QString &title, const QString &iconName)
```

Comment:
You can probably remove this now that @dleacock AMMainWindow changes are tested and approved.